packages/ragbits-document-search/src/ragbits/document_search/documents/element.py

```diff
@@ -90,23 +90,27 @@ def from_vector_db_entry(cls, db_entry: VectorStoreEntry) -> "Element":
         """
         element_type = db_entry.metadata["element_type"]
         element_cls = Element._elements_registry[element_type]
+        if "embedding_type" in db_entry.metadata:
+            del db_entry.metadata["embedding_type"]
         return element_cls(**db_entry.metadata)
 
-    def to_vector_db_entry(self, vector: list[float]) -> VectorStoreEntry:
+    def to_vector_db_entry(self, vector: list[float], embedding_type: EmbeddingTypes) -> VectorStoreEntry:
         """
         Create a vector database entry from the element.
 
         Args:
             vector: The vector.
+            embedding_type: EmbeddingTypes.TEXT, EmbeddingTypes.IMAGE
```

Comment:
Listing specific `EmbeddingTypes` here is not a good practise.

1. What will happen when we add more values to `EmbeddingTypes`? Nobody will remember to update `to_vector_db_entry`'s documentation.
2. This is not a human-readable documentation of the parameter